tensorflow/doc/Tensorflow.md

```diff
@@ -1 +1,30 @@
 # Tensorflow documentation
+
+MLDB incorporates the
+[Tensorflow deep learning framework](https://www.tensorflow.org/).
+This allows MLDB to apply and train deep learning models.
+
+The following functionality is currently exposed in MLDB:
+
+* Application of serialized Tensorflow graphs as a function
+  (see the [Tensorflow Graph Function](TensorflowGraph.md)
+* Access to the full set of Python wrappers for tensorflow.  This
+  allows the examples in Tensorflow to be applied unmodified from within
```

Comment:
Hmmm, you're right.  We would need to set the PYTHONPATH for ipython to make this accessible from a notebook.